### ISSUE TITLE
DetailsList column header sorting order is not Reading in NVDA

### ISSUE THREAD

**Issue by sagoenka** [OfficeDev/office-ui-fabric-react#9092]
Description

Here we used Fabric.DetailsList, the sorting order is not mentioning for every click of space on column Headers of DetailsList in NVDA Screen Reader.
I attached code snippet, please check it,.
Actual behavior:

For Fabric.DetailsList we have columns when we navigate to each column header through clicking on Tab button ,once Tab entered to the DetailsList then i used down Arrow key to navigate to the column Headers and after clicking space on column header, grid is sorting but not reading everytime by NVDA when we click as sorting in Ascending order or Descending order, NVDA reading sometimes only and sorting order is changing every click of space.
Expected behavior:

NVDA should read Sorting order(Ascending/Descending) of column for every click of space on column header of Fabric.DetailsList.
SortingOrderNotMentioning_NVDA.txt

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: NA

**Comment by sagoenka** [comment]
This is a duplicate of https://github.com/OfficeDev/office-ui-fabric-react/issues/8676. opening a new bug with more details.

**Comment by sagoenka** [comment]
While navigating to each column header, NVDA is not reading as 'Press space to sort the column header'(Except for 'File type' column header) and when when user presses enter on the header to sort NVDA is not reading the action performed as 'sorted Ascending or sorted Descending'

https://us-api.asm.skype.com/v1/objects/0-eus-d6-8d9762572895ddbcce75c87383f5fb72/views/imgpsh_mobile_save

https://us-api.asm.skype.com/v1/objects/0-eus-d7-1e13795474653ffac1fc95b5adc21ec8/views/imgpsh_mobile_save

**Comment by KevinTCoughlin** [comment]
We've been getting this report quite often, seems like we should consider a mitigation. The problem with this bug is that the existing W3 ARIA examples do not announce sort state change either, at least since I last checked.

See Example 2 here https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html#Example2

Because of the above, I don't have clear guidance on how to solve / if it should be solved.

**Comment by JasonGore** [comment]
I have also confirmed NVDA is showing this issue with the [ARIA example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html#Example2). Since we are using [`aria-sort` per the ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#aria-sort) and the ARIA example is also exhibiting this issue I would recommend contacting NVDA to resolve this issue.

**Comment by ayrtonsoftware** [comment]
I am also experiencing this issue and would be glad to help track it down...
